ethcore/src/spec/spec.rs

```diff
@@ -98,6 +98,8 @@ pub struct CommonParams {
 	pub block_reward: U256,
 	/// Registrar contract address.
 	pub registrar: Address,
+	/// Apply reward
+	pub apply_reward: bool,
```

Comment:
IMO it is ok and keeps spec structure simple. Having `block_reward` and `apply_reward` in the spec just adds more confusion. You can make a 
`pub fn apply_reward(&self) -> bool` in the code and use that everywhere in case the logic changes.